public/data/arefathi.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Arefat",
+  "type": "Personal",
+  "bio": "Developer | Blogger | Crypto Enthusiast | Reader",
+  "avatar": "https://github.com/arefathi.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/arefathi",
+      "icon": "github"
+    },
+    {
+      "name": "",
+      "url": "",
+      "icon": ""
+    }
+  ],
+  "milestones": [
```

Comment:
If you are not adding anything in milestones section, then you can remove this section.